SYCL/KernelAndProgram/build-log.cpp

```diff
@@ -16,13 +14,23 @@
 //===--------------------------------------------------------------===//
 
 #include <CL/sycl.hpp>
+SYCL_EXTERNAL
+void symbol_that_does_not_exist();
 
 void test() {
   cl::sycl::queue Queue;
 
   // Submitting this kernel should result in a compile_program_error exception
-  // with a message indicating "Unrecognized build options".
-  auto Kernel = []() {};
+  // with a message indicating "CL_BUILD_PROGRAM_FAILURE".
+  auto Kernel = []() {
+#ifdef __SYCL_DEVICE_ONLY__
+#ifdef GPU
+    asm volatile("undefined\n");
```

Comment:
Will it be 'understood' by CUDA compiler?

Reply:
No
These tests were XFAILed on CUDA previously and is still XFAILed.